lib/jake.js

```diff
@@ -151,14 +151,14 @@ if (!global.jake) {
         }
 
         //name = '\033[32m' + p + '\033[39m ';
-        name = chalk.green(p);
+        name = pc.green(p);
 
         descr = task.description;
         if (descr) {
-          descr = chalk.gray('# ' + descr);
+          descr = pc.gray('# ' + descr);
 
           // Create padding-string with calculated length
-          padding = (new Array(maxTaskNameLength - p.length - taskParams.length + 4)).join(' ');
+          padding = ' '.repeat(maxTaskNameLength - p.length - taskParams.length + 4);
```

Comment:
This is the line that I changed in order to avoid the unnecessary array allocation. If that isnt wanted, just accept the suggestion to undo it:

```suggestion
          padding = (new Array(maxTaskNameLength - p.length - taskParams.length + 4)).join(' ');
```